src/main/java/com/example/petbuddybackend/dto/address/AddressDTO.java

```diff
@@ -1,14 +1,17 @@
 package com.example.petbuddybackend.dto.address;
 
 import com.example.petbuddybackend.entity.address.Voivodeship;
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
 
+@Builder
 public record AddressDTO(
     Long id,
```

Comment:
Byłbym dalej za tym, żeby w requestach nie dawać niepotrzebnych pól. Będzie to mylące w dokumentacji a to jest tylko stworzenie nowego obiektu

Reply:
usunalem ID